test/WebSites/RazorWebSite/Controllers/PartialViewEngineController.cs

```diff
@@ -42,5 +42,15 @@ public IActionResult PartialWithModel()
             };
             return PartialView(model);
         }
+
+        public IActionResult RenderPartialWithModel()
```

Comment:
+1 with what @pranavkm said. `GetAwaiter().GetResult()` isn't likely to be host-dependent. As long as the unit tests run under both full CLR and .NET Core, you're covered.